Component/Pet/Pet.js

```diff
@@ -2,30 +2,67 @@ import styled from "styled-components";
 
 const StyledPet = styled.div`
   display: flex;
-  border: 2px solid #000000;
+  border: 5px solid #000000;
   border-radius: 10px;
-  margin: 10px;
-  padding: 10px;
-  justify-content: space-around;
+  margin: 2% 0%;
```

Comment:
If the value is 0, the unit can be omitted.
```suggestion
  margin: 2% 0;
```